docs/api/api-specification.md

```diff
@@ -1456,6 +1577,66 @@ The `http://saraalert.org/StructureDefinition/enrolled-patient` extension is use
 }
 ```
 
+### POST `[base]/Immunization`
+
+<a name="create-post-immunization"/>
+
+To create a new monitoree vaccination, simply POST a FHIR Immunization resource that references the monitoree.
+
+**Request Body:**
+<details>
+  <summary>Click to expand JSON snippet</summary>
+  <div markdown="1">
+
+```json
+{
+  "id": 32,
+  "meta": {
+    "lastUpdated": "2021-04-01T22:09:11+00:00"
+  },
```

Comment:
Remove id and meta from POST example request body.

Reply:
Fixed in https://github.com/SaraAlert/SaraAlert/pull/797/commits/0c7bfdb96f9a95bcba17e573c31942f966608f4a.